### PR TITLE
Improved testing coverage for trace writer

### LINKED ISSUE #47: Add unit testing for trace-writer

### PR DESCRIPTION
**Pull request by matthewloring**
Fixes #47

### CODE REVIEW

**Comment by matthewloring** [comment]
@ofrobots PTAL

File: test/standalone/test-trace-writer.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+var assert = require('assert');
+var nock = require('nock');
+var cls = require('../../lib/cls.js');
+var agent = require('../..');
+var request = require('request');
+
+nock.disableNetConnect();
+
+var uri = 'https://cloudtrace.googleapis.com';
+var path = '/v1/projects/0/traces';
```
Comment (ofrobots) [inline test/standalone/test-trace-writer.js]: Why require a GCLOUD_PROJECT_NUM, but submit traces to project '0'?

Comment (matthewloring) [inline test/standalone/test-trace-writer.js]: Yup, that was a last minute change to require the environment set. It has been fixed.

**Comment by ofrobots** [comment]
LGTM.

**Comment by matthewloring** [comment]
@ofrobots PTAL

**Comment by ofrobots** [comment]
LGTM.

**Pull request merged**